packages/page-staking/src/Performance/AlephBFTCommitteeList.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017-2023 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import { AddressSmall, Table } from '@polkadot/react-components';
+import { useAlephBFTCommittee } from '@polkadot/react-hooks';
+
+import { useTranslation } from '../translate.js';
+
+type Props = {
+  session: number;
+};
+
+const AlephBFTCommitteeList = ({ session }: Props) => {
+  const { t } = useTranslation();
+
+  const committeeAddresses = useAlephBFTCommittee(session);
+
+  const header: [string][] = [
+    [t('Finality committee')]
+  ];
```

Comment:
Better without assertion:
```suggestion
  const header = [
    [t('Finality committee')]
  ] as const;
```
(as long as `<Table>` accepts it of course, but I think it will)

Reply:
I don't think it does - the type is then `readonly` :cry:

Reply:
It is - sometimes it doesn't bother the consumers, but sometimes it indeed does. `as const` is generally preferred over assertions, since assertions can distort types, but if it doesn't fit in a particular case, then no stress about it.

Reply:
Is what I did an assertion? I'm not very familiar with terminology, but I would say I only made `header` type explicit :thinking: 
It does "hide" the header value behind interface(?) :+1:

Reply:
Actually I think it's me using a wrong word for it, sorry. I meant this _manual type definition without inferring_... thing :P Don't worry about it any way, it was a minor suggestion. And yes, if the type defined manually is more suitable in a particular case than the actual type of the variable, then it's fine.